src/styles/blocks/about.scss

```
@@ -0,0 +1,49 @@
+.about-us {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  &__title {
+    font-size: 48px;
+    line-height: 48px;
+    font-weight: 700;
+    letter-spacing: -1.5px;
+    color: #000;
+
+    &-filter {
+      color: #0075ff;
```

```suggestion
      color: #0075ff;
```
Let's make a variable for this color. You use it two often. Actually, it would be good to create variables for all colors